src/main.rs

```diff
@@ -130,8 +130,13 @@ async fn display_time_entries() -> ResultWithDefaultError<()> {
         ),
         Ok(time_entries) => time_entries
             .iter()
+            .take(if count.is_positive() {
+                count as usize
+            } else {
+                time_entries.len()
+            })
```

Comment:
```suggestion
            .take(count.unwrap_or(usize::max_value())
```

Reply:
Should the default be the `time_entries.len()` so it's never bigger than the iterator's length?

Reply:
bah that doesn't seem to work anyway, applied ✅